recipes/dmgbuild/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dmgbuild" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7ced2603d684e29c22b4cd507d1e15a1907e91b86259924b8cfe480d80553b43
+
+build:
+  number: 0
```

Comment:
```suggestion
  number: 0
  noarch: python
```

Reply:
When I used, this caused a warning https://github.com/conda-forge/staged-recipes/pull/26775#issuecomment-2193968539

Reply:
B/c you did not had the conda-forge.yml before.